src/test/java/com/stripe/functional/TimeoutTest.java

```diff
@@ -1,23 +0,0 @@
-package com.stripe.functional;
```

Comment:
restarting that I think we should find a way to test this. Maybe stripe-mock could have an endpoint that explicitly waits X seconds to respond (as a parameter) and we'd put a timeout that is X - 1 ?